test/e2e/storage/vsphere/vsphere_common.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vsphere
+
+import "os"
+
+var (
+	SPBMPolicyName = os.Getenv("VSPHERE_SPBM_POLICY_NAME")
+	StorageClassDatastoreName = os.Getenv("VSPHERE_DATASTORE")
+	SecondSharedDatastore = os.Getenv("VSPHERE_SECOND_SHARED_DATASTORE")
+	KubernetesClusterName = os.Getenv("VSPHERE_KUBERNETES_CLUSTER")
+	SPBMTagPolicy = os.Getenv("VSPHERE_SPBM_TAG_POLICY")
+)
+
+
+var (
+	VCPClusterDatastore = os.Getenv("CLUSTER_DATASTORE")
+	SPBMPolicyDataStoreCluster = os.Getenv("VSPHERE_SPBM_POLICY_DS_CLUSTER")
+)
+
+var (
+	VCPScaleVolumeCount= os.Getenv("VCP_SCALE_VOLUME_COUNT")
+	VCPScaleVolumesPerPod = os.Getenv("VCP_SCALE_VOLUME_PER_POD")
+	VCPScaleInstances = os.Getenv("VCP_SCALE_INSTANCES")
+)
+
+var (
+	VCPStressInstances = os.Getenv("VCP_STRESS_INSTANCES")
+	VCPStressIterations = os.Getenv("VCP_STRESS_ITERATIONS")
+)
+
+var (
+	VCPPerfVolumeCount = os.Getenv("VCP_PERF_VOLUME_COUNT")
+	VCPPerfVolumesPerPod = os.Getenv("VCP_PERF_VOLUME_PER_POD")
+	VCPPerfIterations = os.Getenv("VCP_PERF_ITERATIONS")
+)
```

Comment:
Would be beneficial (less duplicated code perhaps?) to add error handling in this file? IOW handle the cases of a `os.Genenv` error and the case of the variable being empty or not in an expected range?

Reply:
Added error handling in this file.